diesel/src/expression/expression_methods/mod.rs

```diff
@@ -7,11 +7,15 @@
 pub mod bool_expression_methods;
 pub mod escape_expression_methods;
 pub mod global_expression_methods;
+#[cfg(feature = "postgres")]
+pub mod sort_expression_methods;
 pub mod text_expression_methods;
 
 pub use self::bool_expression_methods::BoolExpressionMethods;
 pub use self::escape_expression_methods::EscapeExpressionMethods;
 pub use self::global_expression_methods::ExpressionMethods;
+#[cfg(feature = "postgres")]
+pub use self::sort_expression_methods::Ordering;
```

Comment:
Can we put this trait in the `pg` module? I think we can implement this for sqlite as well by generating `expr is not null, expr` for nulls first and `expr is null, expr` for nulls last.